idunn/api/instant_answer.py

```diff
@@ -181,7 +181,7 @@ async def get_instant_answer_intention(intention, query: str, lang: str):
             maps_frame_url=maps_urls.get_places_url(intention.filter, no_ui=True),
         )
 
-    return build_response(result, query=query, lang=lang)
+    return result
```

Comment:
This change still breaks the instant answer in some cases. For example on `?q=restaurant`, this function will return a 204 `Response`, and `build_response()` will fail.
It would probably be clearer to annotate `get_instant_answer_single_place` and `get_instant_answer_intention` to show that the return type is always `Response`. 
Anyway, it's no longer related to this PR, so maybe this change should be reverted for now, and be handled separately.